src/test/java/teammates/test/cases/browsertests/InstructorFeedbackEditPageUiTest.java

```diff
@@ -901,6 +901,7 @@ private void testResponseRate() {
 
         // Change the feedback path of the question and save
         feedbackEditPage = getFeedbackEditPage();
+        assertTrue(verifyIfVisibilityOptionsDivHasAlertClassEnabled());
```

Comment:
Maybe add a comment explaining why we're​ expecting​ the alert class for this particular question.